third-party/py/twitter-commons/src/python/BUCK

```diff
@@ -2,9 +2,6 @@ python_library(
   name = 'twitter-commons',
   base_module = '',
   srcs = glob(['**/*.py']),
-  deps = [
-    '//third-party/py/setuptools:pkg_resources',
-  ],
```

Comment:
pkg_resources is part of a "pex stdlib" (i.e. every pex made by buck will include pkg_resources) so it's an implicit dependency of every python_library. It just creates two copies of pkg_resources in the resulting python_binary which doesn't end well.